CRADLE/CallPeak/callPeak.py

```diff
@@ -30,46 +40,48 @@ def mergePeaks(peakResult):
 	mergedPeak.append(peakResult[0])
 	resultIdx = 0
 	if len(peakResult) == 1:
-		minPValue = np.min(pvalues)
-		if minPValue == 0:
-			mergedPeak[resultIdx][4] = np.nan
-		else:
-			mergedPeak[resultIdx][4] = np.round((-1) * np.log10(minPValue), 2)
-
-		minQValue = np.min(qvalues)
-		if minQValue == 0:
-			mergedPeak[resultIdx][5] = np.nan
-		else:
-			mergedPeak[resultIdx][5] = np.round((-1) * np.log10(minQValue), 2)
+		mergedPeak[resultIdx][4] = updatePvalue(pvalues)
+		mergedPeak[resultIdx][5] = updateQvalue(qvalues)
```

Comment:
See later comment about these functions for context here.
```suggestion
		mergedPeak[resultIdx][4] = newFC(pvalues)
		mergedPeak[resultIdx][5] = newFC(qvalues)
```

Reply:
```suggestion
		mergedPeak[resultIdx][4] = takeMinusLog(pvalues)
		mergedPeak[resultIdx][5] = takeMinusLog(qvalues)
```